Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -2654,6 +2655,22 @@ inline std::optional<PositionArea> BuilderConverter::convertPositionArea(Builder
     };
 }
 
+inline OptionSet<PositionVisibility> BuilderConverter::convertPositionVisibility(BuilderState& builderState, const CSSValue& value)
+{
+    if (value.valueID() == CSSValueAlways)
+        return { };
+
+    auto maybeList = requiredListDowncast<CSSValueList, CSSPrimitiveValue>(builderState, value);
+    ASSERT(maybeList);
+    auto list = *maybeList;
```

Comment:
you should test and bail for every required*Downcast. 
```
if (!maybeList)
    return { };
```
This avoid Typed OM fuzzer crashes (where weird CSSValues may be constructed outside of the parsing code).

Reply:
Makes sense, I was under the impression only `CSSValues` from the parser can get to this method but apparently not.